skrub/_datetime_encoder.py

```diff
@@ -38,6 +38,7 @@ def _is_pandas_format_mixed_available():
 def to_datetime(
     X,
     errors="coerce",
+    random_state=None,
```

Comment:
i'm wondering if we should make it seeded by default, and even maybe not expose the random state

Reply:
In scikit-learn, you always have a `random_state` parameter for testing purposes and reproducibility. But we might not always want to be consistent with scikit-learn; I don't have a strong opinion here.

Reply:
I don't either. we can discuss at the next meeting which generators should be seeded by default. In the meanwhile LGTM!